### DISCUSSION TITLE
Webs are guh

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#15389] [Ideas and suggestions]
- One web can slow down sub from 100km/h to 0 like a brick wall
- They are limiting speed to 1m/s i believe, which is 2 times slower than a crawler
- If you are swimming away from monsters and just barely touch this - you are dead
- No engine on vanilla subs can beat slowdown force
- They often grow in clusters and their triggers overlap summing up the force
- The greater the mass of the sub - the stronger is slowdown, which is a joke
- Speed clamping is counterintuitive: moving straight down is faster than moving diagonal, i would expect that if i turn on the engine and add the weight of the sub i would apply more total force to get out, but no, speed is clamped to 1 by magnitude, so because some of my speed is now in x direction my y speed will be smaller

Why is this weed so op? Why it's not tearable/nukeable? Why does it resist vertical movement as much as horizontal movement, even though it's just hanging?

It's super annoying and not very realistic

**Comment by kaedepon** [comment]
You can prevent it from sinking when it can hang on a weed.
I think this ignores the laws of physics given the weight of subs.

I agree that it is annoying too : >

**Comment by A2ur3** [comment]
Agreed. This change isn't even in the changelog, and there's no way to spot webs on the sonar. It's not fun at all

**Comment by SomeRandomNoobKekeke** [comment]
Some temporary / permament solution i've made back then 
https://steamcommunity.com/sharedfiles/filedetails/?id=3385992492
<img width="519" height="160" alt="BaroDev_ultra_wide" src="https://github.com/user-attachments/assets/9a0d715c-c34d-4bae-96e1-72876fd7420d" />
